lib/daemon_runner/retry_errors.rb

```diff
@@ -0,0 +1,18 @@
+require 'retryable'
+
+module DaemonRunner
+    #
+    # Retry Errors
+    #
+    class RetryErrors
+      class << self
+        def retry(retries: 3, exceptions: [Faraday::ClientError], &block)
+          Retryable.retryable({
+            on: exceptions,
```

Comment:
What does that do? As far as I can see, you're just calculating the same power every time. Should the retry wrapper continue to do this forever, or eventually throw an exception farther up stack?